src/shared/varlink-io.systemd.Machine.c

```diff
@@ -4,6 +4,13 @@
 
 #include "varlink-io.systemd.Machine.h"
 
+#define SD_VARLINK_LOOKUP_AND_POLKIT_INPUT_FIELDS                                                                                              \
+        SD_VARLINK_FIELD_COMMENT("If non-null the name of a machine."),                                                                        \
+        SD_VARLINK_DEFINE_INPUT(name, SD_VARLINK_STRING, SD_VARLINK_NULLABLE),                                                                 \
+        SD_VARLINK_FIELD_COMMENT("If non-null the PID of a machine. Special value 0 means to take pid of the machine the caller is part of."), \
+        SD_VARLINK_DEFINE_INPUT(pid, SD_VARLINK_INT, SD_VARLINK_NULLABLE),                                                                     \
+        SD_VARLINK_DEFINE_INPUT(allowInteractiveAuthentication, SD_VARLINK_BOOL, SD_VARLINK_NULLABLE)
```

Comment:
btw, this should use the newly added VARLINK_DEFINE_POLKIT_INPUT macro now.

see e47dbf5b4e2288ea8c8c930f43c8f4897c586d36